docs/developer-guide/shareable-configs.md

```diff
@@ -148,7 +148,6 @@ myconfig
 ├── index.js
 └─┬ lib
   ├── defaults.js
-  ├── dev.js
   ├── ci.js
```

Comment:
I believe the idea was to present a config package with different configurations for development and CI as well as different configurations for backend and frontend (which also isn't used later in the text), while providing examples of the content for only some of the files.

I'd personally leave this structure as is.

Reply:
Ah, I understand now. I didn't even notice that `frontend.j` wasn't in any of the examples either.

I think we can leave it as is for now. Maybe if someone else indicates `dev.js` and `frontend.js` without examples is confusing, then we can either provide examples (which would just add more wording unless they can also demonstrate a new concept the existing examples don't already illustrate) or add a paragraph just stating they (`dev.js` and `frontend.js`) don't have corresponding examples and are just to help add clarification to a possible shareable configuration directory structure.